wiki/Beatmap_Editor/Timelines/en.md

```diff
@@ -1,137 +1,141 @@
+### osu!, osu!taiko, and osu!catch
 
-![Hit Objects Timeline](/wiki/shared/BE_NTL.jpg "Hit Objects Timeline")
+![Hit Objects Timeline in osu!, osu!taiko, and osu!catch](/wiki/shared/BE_NTL.jpg "This shows the timeline for hit objects with respect to the beat snap divisor and timestamp.")
 
-**Shows the timeline for hitobjects with respect to [Beat Snap Divisor](/wiki/Beatmap_Editor/Beat_Snap_Divisor) and timestamp.**
-
-#### Hit Object Timeline
+In compose mode, this timeline is underneath the [Compose](/wiki/Compose) tab for any game mode except for [osu!mania](wiki/osu!mania).
 
 | Name | Description |
 | ---- | ----------- |
-| `+`/`-` buttons | Increase/Decrease timeline zoom |
-| Double white vertical lines | Current timestamp with respect to the Hit Objects Timeline. |
-| Click on hitobject | **Left-Click:** Select / Move hitobject against timeline. **Right-Click:** Remove hitobject. |
+| `+`/`-` buttons | Increase/Decrease timeline zoom. |
+| Double white vertical lines | This shows the current timestamp with respect to the hit object timeline. |
 
-### osu!mania
+Left clicking on a hit object will select it and dragging will move the selected hit object against timeline.
 
-*Location: Centre @ [Compose](/wiki/Beatmap_Editor/Compose) tab in osu!mania only.*
+Right clicking will remove the selected hit objects.
+
+### osu!mania
 
 ![osu!mania playfield](/wiki/shared/BEM_PF.jpg "osu!mania playfield")
 
-#### Playfield Boxes
+In compose mode, this timeline is in the centre of the playfield for osu!mania mode.
 
-| Name | Description |
-| ---- | ----------- |
-| Left Box | Notes intensity (Timeline). |
-| Centre | Actual Playfield ([Beat Snap Divisor](/wiki/Beatmap_Editor/Beat_Snap_Divisor)). |
+The box on the left is a horizontal bar char that displays the note intensity.
+This acts like a timeline.
 
-#### Colour notes
+In the centre is the actual playfield.
+The playfield is made up of two parts: lines and notes.
 
-| Name | Description |
+| Line Colour | Description |
 | ---- | ----------- |
-| Double white line | Measure |
-| White line | Common Time |
-| Green line | You / Judgement Line |
-| Blue colour | Currently-selected note |
-| White/Pink/Yellow colour | Normal notes colour |
+| Thick White | Full measure |
+| White | Common Time |
+| Green | Current timestamp/Judgement line |
 
-Design
---------
+| Note Colour | Description |
+| ---- | ----------- |
+| Blue | Selected note(s) |
+| White/Pink/Yellow | Unselected note colours |
 
-*Location: Top @ [Design](/wiki/Beatmap_Editor/Design) tab*
+## Design
 
-**Shows the timeline for transformations of the selected object.**
+![Storyboard Timeline](/wiki/shared/SE_STM.jpg "This shows the timeline for transformations of the selected sprite.")
 
-![Storyboard Timeline](/wiki/shared/SE_STM.jpg "Storyboard Timeline")
+The design timeline is located underneath the [Design](/wiki/design) tab.
```

Comment:
`[`Design`](/wiki/design) tab.` ? you don't have to accept this one or the compose one, idk